src/helm/common/request.py

```diff
@@ -15,8 +14,12 @@ class Request:
     various APIs (e.g., GPT-3, Jurassic).
     """
 
+    model_deployment: str = ""
+    """Which model to query.
+    Refers to a deployment in the model deployment registry."""
+
     model: str = "openai/text-davinci-002"
-    """Which model to query"""
+    """DEPRECATED. Use `model_deployment` instead."""
```

Comment:
I don't think you can deprecate this? I think you have _both_ `model` and `model_deployment` set. They are different concepts.

Then you can revert the various places where you use `request.effective_model_deployment` - revert to using `request.model` instead

Reply:
see answer above.

Reply:
After our conversation, it's not deprecated anymore